src/styles/body-blocks/stars.scss

```diff
@@ -0,0 +1,18 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
+    line-height: 140%;
+  }
+  @for $starCount from 1 through 5 {
+    &--#{$starCount} .stars__star:nth-child(-n + #{$starCount}) {
+      background-image: url(../images/star-active.svg);
+    }
+  }
```

Comment:
1) space before the loop and `}` 
2) Why not just use `$i`, it is common to name iterators this way. (But your solution is not inherently wrong)